client-spark/spark3/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -125,22 +127,31 @@ private synchronized void putBlockId(
 
   public RssShuffleManager(SparkConf conf, boolean isDriver) {
     this.sparkConf = conf;
+
+    // set & check replica config
+    this.dataReplica = sparkConf.getInt(RssClientConfig.RSS_DATA_REPLICA,
+      RssClientConfig.RSS_DATA_REPLICA_DEFAULT_VALUE);
+    this.dataReplicaWrite =  sparkConf.getInt(RssClientConfig.RSS_DATA_REPLICA_WRITE,
+      RssClientConfig.RSS_DATA_REPLICA_WRITE_DEFAULT_VALUE);
+    this.dataReplicaRead =  sparkConf.getInt(RssClientConfig.RSS_DATA_REPLICA_READ,
+      RssClientConfig.RSS_DATA_REPLICA_READ_DEFAULT_VALUE);
+
```

Comment:
check quorum config here immediately

Reply:
When moving check to RssShuffleManager, we have to maintain this for both spark2 and spark3.
Therefore, I check the config in the constructor of ShuffleWriteClientImpl because them are really used in this place.
@duanmeng

Reply:
Quorum config checking is part of your ShuffleManager initialization that is maintained for both spark2 and spark3. It's odd to check all them in a write client as they are end to end quorum configurations.

Reply:
ok.
It has been updated.